src/mutations/member.ts

```diff
@@ -146,10 +146,14 @@ export default (queryClient: QueryClient, queryConfig: QueryClientConfig) => {
       const newFavoriteItems = prevExtra.favoriteItems
         ? prevExtra.favoriteItems.concat([itemId])
         : [itemId];
+      const uniqueFavoriteItems = [...new Set(newFavoriteItems)];
```

Comment:
Any reason why we don't store the favorite items as set directly? I'm fine with making "immutable" copies, but I'm not sure about the cost of converting back and forth between sets and arrays. From a modeling perspective it would also make more sense to represent it as a set.